c2corg_ui/static/js/apiservice.js

```diff
@@ -350,4 +359,38 @@ app.Api.prototype.updateAccount = function(data) {
 };
 
 
+
+/**
+ * @param {Object} file
```

Comment:
If your method takes a `File`, you must:
- declare a `File` param;
- call it with a `File` parameter.
This should be the case for this method and be the general principle for all methods in the project.